### PR TITLE
Hw15 (docker)

### PR DESCRIPTION
**Pull request by Engelko**

### CODE REVIEW

File: Bash_scripts/HW15/README.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Dockerfile syntax:
+1.	Difference between ARG and ENV
+2.	What is VOLUME? Create an example of a Dockerfile using VOLUME
+3.	Create NOBODY user and run container as this user
```
Comment (mavrik90) [inline Bash_scripts/HW15/README.md:4]: User should be created inside your Dockerfile

Comment (Engelko) [reply Bash_scripts/HW15/README.md:4]: hw15p3_change


Was that meant in the assignment?

Comment (mavrik90) [reply Bash_scripts/HW15/README.md:4]: This is correct

**Review by mavrik90 — APPROVED** [review]
LGTM

**Pull request merged**